Dockerfile

```
@@ -379,6 +379,9 @@ COPY --from=machined-build-amd64 /machined /rootfs/sbin/init
 COPY ./hack/cleanup.sh /toolchain/bin/cleanup.sh
 RUN cleanup.sh /rootfs
 RUN mkdir -pv /rootfs/{boot,etc/cri/conf.d/hosts,usr/local/share,mnt,system,opt}
+RUN mkdir -pv /rootfs/usr/share/zoneinfo/Etc
+COPY --chmod=0644 hack/zoneinfo/Etc/UTC /rootfs/usr/share/zoneinfo/Etc/UTC
+RUN ln -s /etc/localtime /rootfs/usr/share/zoneinfo/Etc/UTC
```

I guess it should be: (not tested)

```suggestion
RUN ln -s /rootfs/etc/localtime /usr/share/zoneinfo/Etc/UTC
```